src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,54 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 10;
+    private K[] keys;
+    private V[] values;
+    private int size;
+
+    public StorageImpl() {
+        keys = (K[]) new Object[MAX_ITEMS_NUMBER];
+        values = (V[]) new Object[MAX_ITEMS_NUMBER];
+        size = 0;
+    }
+
+    public boolean keyCheck(K keys, K key) {
```

Comment:
Private ones (methods, classes) should be after public ones.
```suggestion
    private boolean isEquals(Object first, Object second) {
```